static/sass/styleguide.scss

```diff
@@ -50,6 +50,13 @@ h4 {
   padding: 0 size('shim');
 }
 
+[data-sassdoc-table='parameter'] {
```

Comment:
Does this need to be fixed in Herman? If so: no need to hold up merge, but an issue should be filed.